src/main/java/io/jenkins/plugins/oidc_provider/config/ClaimTemplate.java

```diff
@@ -44,11 +44,13 @@ public final class ClaimTemplate extends AbstractDescribableImpl<ClaimTemplate>
     public final @NonNull String name;
     public final @NonNull String format;
     public final @NonNull ClaimType type;
+    public final String requiredEnvVars;
 
-    @DataBoundConstructor public ClaimTemplate(String name, String format, ClaimType type) {
+    @DataBoundConstructor public ClaimTemplate(String name, String format, ClaimType type, String requiredEnvVars) {
```

Comment:
As a general rule, we shouldn't modify the signature of a public method or constructor to keep binary compatibility. Could you please add a new constructor leaving there the former one? There's no problem in changing which constructor is annotated

Reply:
Anyway optional fields should use `@DataBoundSetter`.

Reply:
this should be fixed.